recipes/uconfig/all/conanfile.py

```diff
@@ -0,0 +1,80 @@
+from conans import ConanFile, tools
+from conans.errors import ConanInvalidConfiguration
+
+import os
+
+required_conan_version = ">=1.33.0"
+
+
+class UconfigConan(ConanFile):
+    name = "uconfig"
```

Comment:
```suggestion
    name = "tinkoffcreditsystem-uconfig"
```
There are more projects using same name: https://github.com/search?q=uconfig

Robotips/uConfig is so far more popular and is C++ too, I see a name collision here.

Reply:
Renamed into `tcsbank-uconfig`